src/forms/services/form-submission.service.ts

```
@@ -1211,51 +1224,77 @@
-                    for (const oneOfItem of (depSchema as any).oneOf) {
-                      if (oneOfItem.properties) {
-                        extractFieldMappings(oneOfItem.properties, currentPage);
-                      }
-                    }
-                  } else if ((depSchema as any).properties) {
-                    // Handle direct properties dependencies
-                    extractFieldMappings(
-                      (depSchema as any).properties,
-                      currentPage
+                for (const depSchema of Object.values(dependencies)) {
+                  if (!depSchema || typeof depSchema !== 'object') continue;
```

ESLint

[error] Replace `'object'` with `"object"`

(prettier/prettier)